### PR TITLE
Update lib.rs

### PR DESCRIPTION
**Pull request by lightmark**
## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

File: storage/schemadb/src/lib.rs [hunk 1]
```diff
@@ -216,6 +216,7 @@ impl DB {
         let mut db_opts = rocksdb::Options::default();
         db_opts.create_if_missing(true);
         db_opts.create_missing_column_families(true);
+        db_opts.set_max_open_files(1000);
```
Comment (mgorven) [inline storage/schemadb/src/lib.rs:219]: 1024 for nice computer sciencey numbers? ;-)

Comment (mgorven) [reply storage/schemadb/src/lib.rs:219]: Also this should probably be configurable.

Comment (davidiw) [reply storage/schemadb/src/lib.rs:219]: I suspect there are performance considerations here? Though I don't we'd see anything in cluster test.

Comment (lightmark) [reply storage/schemadb/src/lib.rs:219]: Yes. I don't know whether I should land this or I make a refactor to create an option file that can be imported by our db.

Comment (lightmark) [reply storage/schemadb/src/lib.rs:219]: @davidiw theoretically the bigger this number, the better the performance. But I think 1k is good enough compared to inf.

**Comment by lightmark** [comment]
Move to https://github.com/libra/libra/pull/6656

**Pull request closed without merging**